docs/levenscyclus/index.md

```diff
@@ -1,32 +1,37 @@
 --- 
 title: Levenscyclus
+summary: Hulp per fase van de levenscyclus van algoritmes en AI, zoals probleemanalyse, ontwerpen, ontwikkelen, implementeren en ermee stoppen.
 icon: material/reload
 --- 
+# Levenscyclus algoritmes en AI
 
-!!! info "Disclaimer"
+Verantwoordelijke algoritmes en AI voldoen tijdens elke fase van de levenscyclus aan de regels. Van het nemen van een besluit tot het stoppen met algoritmes of AI.
 
-    Het Algoritmekader is nog volop in ontwikkeling. Op deze plek willen we vooral aan de slag gaan op een open en transparante wijze. Het is dus niet definitief. Dat betekent dat er dingen opstaan die niet af zijn en soms zelfs fout. Mocht er iets niet kloppen, laat het ons weten via [GitHub](https://github.com/MinBZK/Algoritmekader).
+## Fases van de levenscyclus
+1.	[Probleem analyseren](probleemanalyse.md)
+2.	[Ontwerpen](ontwerp.md)
+3.	[Data onderzoeken en voorbereiden](dataverkenning-en-datapreparatie.md)
+4.	[Ontwikkelen](ontwikkelen.md)
+5.	[Beoordelen](verificatie-en-validatie.md)
+6.	[Implementeren](implementatie.md)
+7.	[Controle en onderhoud](monitoring-en-beheer.md)
+8.	[Stoppen](uitfaseren.md)
 
-Algoritmes en AI kunnen door overheidsorganisaties worden gebruikt, bijvoorbeeld bij het leveren van diensten of het nemen van besluiten. 
-Algoritmes en AI-systemen doorlopen een zogeheten levenscyclus. 
-Een algoritme wordt ontwikkeld, in productie genomen, en na enige tijd van gebruik kan worden besloten het gebruik ervan te beëindigen. 
-Omdat de levenscyclus van een algoritme en AI aanzet tot nadenken over de inzet van algoritmes van begin tot eind is het bruikbaar als leidraad om relevante informatie te structureren en te communiceren. 
-Wat is het doel, gaat het gebruik ervan wel naar verwachting, wanneer moet wat worden gedaan en wat als het gebruik ten einde loopt? 
-In praktijk kan het zo zijn dat de levenscyclus anders verloopt, bijvoorbeeld omdat na validatie en verificatie eerst terug naar de tekentafel moet worden gegaan (ontwerpfase) omdat het nog niet aan de wensen of vereisten voldoet.  
+De fases in deze levenscyclus passen bij de manier van werken van overheden. Andere fases zijn ook mogelijk.
```

Comment:
dit is een beetje te stellig denk ik. Iedere organisatie werkt anders. Zie suggestie hieronder
```suggestion
```

Reply:
Oeps, ik zie geen verschil tussen jouw en mijn suggestie. Ging hier iets fout? Of mis ik iets?

Reply:
Excuus, was niet duidelijk. Heb het in deze suggestie verwerkt:  https://github.com/MinBZK/Algoritmekader/pull/201/files#r1711825103